pkg/icingadb/history/sla.go

```diff
@@ -21,7 +20,7 @@ func stateHistoryToSlaEntity(entry redis.XMessage) ([]history.UpserterEntity, er
 	}
 	slaState := slaStateInterface.(*history.SlaHistoryState)
 
-	if slaState.StateType != types.StateHard {
+	if slaState.StateType != "hard" {
```

Comment:
I am a bit unhappy with the appearance of the magic strings. Before, there were `const` values to compare against, but now there are "raw" strings, being prone for typos and to be forgotten in a future refactoring.

This is also prominent in the `cmd/icingadb-migrate/convert.go` file.